schedule/yast/raid/testdata/lvm_raid1.yaml

```diff
@@ -0,0 +1,73 @@
+---
+arch: x86_64
+disks:
+  vda:
+    partitions:
+      - id: bios-boot
+        role: raw-volume
+        size: 2mb
+      - id: linux-raid
+        role: raw-volume
+        size: 800mb
+      - id: linux-raid
+        role: raw-volume
+        size: 100mb
+  vdb:
+    partitions:
+      - id: bios-boot
+        role: raw-volume
+        size: 2mb
+      - id: linux-raid
+        role: raw-volume
+        size: 800mb
+      - id: linux-raid
+        role: raw-volume
+        size: 100mb
+  vdc:
+    partitions:
+      - id: bios-boot
+        role: raw-volume
+        size: 2mb
+      - id: linux-raid
+        role: raw-volume
+        size: 800mb
+      - id: linux-raid
+        role: raw-volume
+        size: 100mb
+  vdd:
+    partitions:
+      - id: bios-boot
+        role: raw-volume
+        size: 2mb
+      - id: linux-raid
+        role: raw-volume
+        size: 800mb
+      - id: linux-raid
+        role: raw-volume
+        size: 100mb
+lvm:
+  format: lvm2
+  lvname: root
+  lvpath: /dev/root/root
+  pvname: /dev/md0
+  size: '7.69 GiB'
```

Comment:
Please, remove all data that your test is not using, it is quite confusing. Better make it simple and later extend it.